conda/recipes/libcuspatial/meta.yaml

```diff
@@ -54,16 +58,18 @@ outputs:
     requirements:
       build:
         - cmake {{ cmake_version }}
+        - ninja
       run:
         - cudatoolkit {{ cuda_spec }}
         - libcudf {{ minor_version }}.*
         - librmm {{ minor_version }}.*
+        - libcusparse=11.7.0.107 # This version corresponds to CUDA 11.5
```

Comment:
Previously, gpuCI's images included a full CTK installation. This package build was implicitly using some of the system CTK (cusparse) which is not included in `conda-forge::cudatoolkit`. This package comes from the `nvidia` channel. It isn't exactly ideal to mix `conda-forge::cudatoolkit` with `nvidia::libcusparse` but it's probably no worse than relying on the system CTK as in the previous CI images. (We have similar dependencies on `nvidia::cuda-nvtx` in `libcudf`.)